models/stg_shopify__inventory_item.sql

```diff
@@ -29,11 +29,21 @@ final as (
         id as inventory_item_id,
         sku,
         _fivetran_deleted as is_deleted, -- won't filter out for now
-        cost,
+        coalesce(unit_cost_amount, cost) as unit_cost_amount,
+        unit_cost_currency_code,
         country_code_of_origin,
         province_code_of_origin,
         requires_shipping as is_shipping_required,
         tracked as is_inventory_quantity_tracked,
+        duplicate_sku_count,
+        harmonized_system_code,
+        inventory_history_url,
+        legacy_resource_id,
+        measurement_id,
+        measurement_weight_value,
+        measurement_weight_unit,
+        tracked_editable_locked as is_tracked_editable_locked,
```

Comment:
This is the field currently `tracked_editable_editable` in the connector. Should be fixed at connector level but noting in case we have to update the spelling here.